ring-core/test/ring/middleware/multipart_params/test/multipart_params.clj

```diff
@@ -0,0 +1,27 @@
+(ns ring.middleware.multipart-params.test.multipart-params
```

Comment:
Why have you created a new test file? Just add your tests to the existing file at: `ring-core/test/ring/middleware/test/multipart_params.clj`